@codex please fix

```
Run python desktop-tauri/scripts/test_packaged_operator_e2e.py
Traceback (most recent call last):
  File "/home/runner/work/token.place/token.place/desktop-tauri/scripts/test_packaged_operator_e2e.py", line 188, in <module>
    raise SystemExit(main())
                     ^^^^^^
  File "/home/runner/work/token.place/token.place/desktop-tauri/scripts/test_packaged_operator_e2e.py", line 156, in main
    raise RuntimeError(f"bridge emitted error event: {payload}")
RuntimeError: bridge emitted error event: {'type': 'error', 'message': "bridge failure: No module named 'encrypt'"}
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/.github/workflows/desktop-operator-e2e.yml
+++ b/.github/workflows/desktop-operator-e2e.yml
@@ -9,6 +9,7 @@ on:
       - 'desktop-tauri/scripts/test_packaged_operator_e2e.py'
       - 'utils/**'
       - 'config.py'
+      - 'encrypt.py'
       - 'relay.py'
       - 'requirements.txt'
   push:
@@ -20,6 +21,7 @@ on:
       - 'desktop-tauri/scripts/test_packaged_operator_e2e.py'
       - 'utils/**'
       - 'config.py'
+      - 'encrypt.py'
       - 'relay.py'
       - 'requirements.txt'
 

--- a/desktop-tauri/scripts/test_packaged_operator_e2e.py
+++ b/desktop-tauri/scripts/test_packaged_operator_e2e.py
@@ -64,6 +64,7 @@ def create_packaged_layout(tmp_root: Path) -> Path:
         )
 
     shutil.copy2(REPO_ROOT / "config.py", resources_root / "config.py")
+    shutil.copy2(REPO_ROOT / "encrypt.py", resources_root / "encrypt.py")
     shutil.copytree(REPO_ROOT / "utils", resources_root / "utils", dirs_exist_ok=True)
 
     return python_dir / "compute_node_bridge.py"

--- a/desktop-tauri/src-tauri/src/main.rs
+++ b/desktop-tauri/src-tauri/src/main.rs
@@ -391,6 +391,7 @@ mod tests {
             "python/path_bootstrap.py",
             "../../utils",
             "../../config.py",
+            "../../encrypt.py",
         ];
 
         // Intentional strict count: this guards against accidental bundle bloat.

--- a/desktop-tauri/src-tauri/tauri.conf.json
+++ b/desktop-tauri/src-tauri/tauri.conf.json
@@ -30,7 +30,8 @@
       "python/model_bridge.py",
       "python/path_bootstrap.py",
       "../../utils",
-      "../../config.py"
+      "../../config.py",
+      "../../encrypt.py"
     ],
     "icon": [
       "icons/32x32.png",

